tree-walk_test.go

```diff
@@ -17,76 +17,48 @@
 package main
 
 import (
-	"bytes"
-	"strconv"
+	"fmt"
+	"io/ioutil"
 	"strings"
 	"testing"
 	"time"
 )
 
-// Helper function that invokes startTreeWalk depending on the type implementing objectLayer.
-func startTreeWalk(obj ObjectLayer, bucket, prefix, marker string,
-	recursive bool, endWalkCh chan struct{}) chan treeWalkResult {
-	var twResultCh chan treeWalkResult
-	switch typ := obj.(type) {
-	case fsObjects:
-		twResultCh = typ.startTreeWalk(bucket, prefix, marker, true,
-			func(bucket, object string) bool {
-				return !strings.HasSuffix(object, slashSeparator)
-			}, endWalkCh)
-	case xlObjects:
-		twResultCh = typ.startTreeWalk(bucket, prefix, marker, true,
-			typ.isObject, endWalkCh)
-	}
-	return twResultCh
+// Sample entries for the namespace.
+var volume = "testvolume"
+var files = []string{
```

Comment:
nit: we should keep our jargon consistent. We mean objects but we refer them as files in variable names This is confusing.
e.g ``var files []string``